pkg/node/sendoperation/deploySC/deploySC.go

```diff
@@ -0,0 +1,45 @@
+package deploysc
+
+import (
+	"io"
+
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/massalabs/thyra/api/swagger/server/models"
+	"github.com/massalabs/thyra/api/swagger/server/restapi/operations"
+	"github.com/massalabs/thyra/pkg/node"
+	"github.com/massalabs/thyra/pkg/onchain"
+)
+
+func CreateCmdDeploySCHandler() func(params operations.CmdDeploySCParams) middleware.Responder {
+	//nolint:gocritic
+	return func(params operations.CmdDeploySCParams) middleware.Responder {
+		return cmdDeploySCHandler(params)
+	}
+}
+
+func cmdDeploySCHandler(params operations.CmdDeploySCParams) middleware.Responder {
+	client := node.NewDefaultClient()
+
+	file, err := io.ReadAll(params.Wasmfile)
+	if err != nil {
+		return operations.NewCmdDeploySCInternalServerError().
```

Comment:
has it's an error regarding the parsing of an argument, you should use a 4XX error intead of a 500